### PR TITLE
chore: prettier formatting

### PR DESCRIPTION
**Pull request by sean-perkins**
Runs prettier formatting on the docs changes.

### CODE REVIEW

File: versioned_docs/version-v5/angular/navigation.md
Comment (thetaPC) [inline versioned_docs/version-v5/angular/navigation.md:1]: Was the note removed intentionally?

File: docs/angular/navigation.md
Comment (thetaPC) [inline docs/angular/navigation.md:1]: Was the note removed intentionally?

Comment (sean-perkins) [reply docs/angular/navigation.md:1]: Good call out 🤦 somehow my stashed local changes were applied here. I'll revert those.

**Review by thetaPC — APPROVED** [review]
LGTM

**Pull request merged**